app/utils/fileUrlUtils.ts

```diff
@@ -2,7 +2,7 @@ import { FileType } from "app/models";
 import { Constants } from "app/utils";
 
 export function parseRelativePath(fileUrl: string): FileType {
-    let parts: string[] = fileUrl.split(Constants.FileUrlStrings.Separator);
+    const parts: string[] = fileUrl.split(Constants.FileUrlStrings.Separator);
```

Comment:
I tested with both windows and linux machines. Looks like the URL separator is always consistent.